LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017–2025 Zach Leatherman @zachleat
```

Comment:
The very first commit ( https://github.com/11ty/eleventy-navigation/commit/0fd202e9ac0f8fac951123426bb93d07cc5f46ee ) is from 2019.

Reply:
thank you for the note, update